build/common.props

```diff
@@ -6,7 +6,7 @@
 
   <!-- When building on non-Windows machines, we need the reference assemblies to build full-framework assemblies. -->
   <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework' AND '$(DotNetBuildFromSource)' == 'true'">
-     <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0-alpha-5" PrivateAssets="All" />
+     <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0" PrivateAssets="All" />
```

Comment:
I think this an oversight on my end. 
We should manage the version centrally.

Reply:
Ok, will move the version to packages.targets file

Reply:
Actually, I removed the reference assembly, because it seems it is not required, since source-build is building only netcore for 5.0; see https://github.com/NuGet/NuGet.Client/pull/3067#discussion_r326731073